### PR TITLE
New package: q5go-2.13

### PR DESCRIPTION
**Pull request by jason1987d**
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): YES

- I built this PR locally for my native architecture, (x86_64-libc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl

### CODE REVIEW

**Comment by tibequadorian** [comment]
Please don't create new PRs over and over, it's better to `git commit --amend` and force push your changes: https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#review

**Comment by jason1987d** [comment]
Noted, and my apologies. I have set my default git pull strategy to be rebase to avoid any other issues with merge commits and amendments. Is anything else required for explanation of this package or pull request? 

The older package (~2008) qGo is in void's repos, but this program is a massive upgrade of that using qt5.

File: srcpkgs/q5go/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Qt5 Go/Weiqi/Baduk client modernized from qGo"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/bernds/q5Go"
+distfiles="https://github.com/bernds/q5Go/archive/refs/tags/${pkgname}-${version}.tar.gz"
+checksum=00a685eed2bb4318ce148e916248def63ac484dda6eb8b3cda501a526e215d67
+
+post_install() {
+	qmake -install qinstall "${wrksrc}/src/images/clientwindow/Bowl.png" "${DESTDIR}/usr/share/q5go/Bowl.png"
+	qmake -install qinstall "${wrksrc}/templates/qgo.desktop" "${DESTDIR}/usr/share/applications/q5go.desktop"
```
Comment (tibequadorian) [inline srcpkgs/q5go/template:19]: I there an advantage of this over`vinstall`?

File: srcpkgs/q5go/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	sed -i -e 's/Name=qgo/Name=q5go/g' -e 's/Comment=qgo/Comment=q5go/g' -e 's/Exec=qgo/Exec=q5go/g' -e 's/Icon=\/usr\/share\/qGo\/pics\/Bowl\.png/Icon=\/usr\/share\/q5go\/Bowl\.png/g' -e 's/MimeType=text\/plain\;text\/sfg\;Application/MimeType=application\/x-go-sgf;/g' "${DESTDIR}/usr/share/applications/q5go.desktop"
```
Comment (Chocimier) [inline srcpkgs/q5go/template:20]: Please replace with a patch and send it upstream.

**Comment by jason1987d** [comment]
My branch got messed up, and accidentally closed the PR when cleaning that up. Was able to recover the template.
However, a new version 2.13 of this has released, and I will also address the icon/desktop files as patches and not how I did.

**Comment by jason1987d** [comment]
I have this all fixed now, with patches to q(5)go.desktop

**Comment by jason1987d** [comment]
Also, I will be looking to build a template for software that works well with this KataGo https://github.com/lightvector/KataGo/blob/master/Compiling.md but has multiple backends (OpenCL,CUDA,TensorRT,Eigen) to choose from. When organizing a template, would it be done as a subpackage for each possible backend?

**Comment by Chocimier** [comment]
> would it be done as a subpackage for each possible backend

Likely, let's discuss in dedicated PR.

**Pull request merged**